rep-2000.rst

```diff
@@ -709,6 +709,121 @@ Package manager use for dependencies:
+
+The following indicators show what delivery mechanisms are available for each platform.
+
+" [d] " Debian packages will be provided for this platform for packages submitted to the rosdistro.
+
+" [a] " Binary releases are provided as a single archive per platform containing all packages in the Galactic ROS 2 repos file [10]_.
+
+" [s] " Compilation from source.
+
+Middleware Implementation Support:
+
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------+
+| Middleware Library       | Middleware Provider | Support Level | Platforms                   | Architectures                  |
++==========================+=====================+===============+=============================+================================+
+| rmw_cyclonedds_cpp       | Eclipse Cyclone DDS | Tier 1        | All Platforms               | All Architectures              |
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------+
+| rmw_fastrtps_cpp         | eProsima Fast-DDS   | Tier 1        | All Platforms               | All Architectures              |
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------+
+| rmw_connext_cpp          | RTI Connext         | Tier 1        | All Platforms except Debian | All Architectures except arm64 |
+|                          |                     |               | and OpenEmbedded            |                                |
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------+
+| rmw_fastrtps_dynamic_cpp | eProsima Fast-DDS   | Tier 2        | All Platforms               | All Architectures              |
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------+
+
+Middleware implementation support is dependent upon the platform support tier.
+For example a Tier 1 middleware implementation on a Tier 2 platform can only receive Tier 2 support.
+
+Minimum language requirements:
+
+- C++17
+- Python 3.7
+
+
+Dependency Requirements:
+
++-------------+---------------------------------+------------------------------------------------------+
+|             |        Required Support         |                 Recommended Support                  |
++-------------+----------------+----------------+---------------+----------------+---------------------+
+| Package     |  Ubuntu Focal  |   Windows 10** |     MacOS**   | Debian Buster  | OpenEmbedded**      |
++=============+================+================+===============+================+=====================+
+| CMake       |     3.16.3     |     3.14.4     |     3.14.4    |      3.13.4    | 3.16.1 / 3.12.2**** |
```

Comment:
For the Windows dependencies, does it make sense to bump them all to the current available versions (at the time of publishing this document)? E.g. I presume the version of CMake we're installing is at least the version used on Ubuntu Focal.

Reply:
Good point.  We should probably update this to the versions that are currently being used in CI.  As we get closer to the release, and/or we update the dependencies, we can update this table.

Reply:
This was a great point - we were WAY off. I've updated the version numbers in 8ac5876.